src/utils/utils.js

```diff
@@ -295,8 +295,25 @@ export const getconstructorParams = (abiConstructor, state, vals, crowdsaleNum)
     return params;
 }
 
-export function roundToDecimals(n, input) {
-  return 1.0 / n * Math.ceil(n * input)
+export const roundToDecimals = (n, input) => Math.floor10(input, n)
+
+const decimalAdjust = (type, value, exp) => {
+    if (typeof exp === 'undefined' || +exp === 0) {
+      return Math[type](value);
+    }
+    value = +value;
```

Comment:
why all the '+value's? There should be a clearer way of accomplishing your intent here.

Reply:
Let's use  Math.floor10 and decimalAdjust. It is well-known solution: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round

Reply:
I agree. All the '+' signs in the expressions will be replaced by Math.round? Or other methods on the Math object?

Reply:
it is the numeric representation: https://stackoverflow.com/questions/6682997/javascript-plus-symbol-before-variable